CONTRIBUTING.md

```diff
@@ -161,6 +161,9 @@ To build all libraries and tests, with Python packages in development mode, simp
 ./build.sh --pydevelop libcudf libcudf_kafka cudf dask_cudf cudf_kafka custreamz
 ```
 
+- **WARNING**: Various changes to files, such as .pyx and .pyd, may cause errors such as
+  circular imports until you build again.
```

Comment:
@vyasr may have opinions on where this should go (maybe the [developer guide](https://docs.rapids.ai/api/cudf/stable/developer_guide/)?) -- but in the meantime, I'd propose this simplification:
```suggestion
Note that if Cython files (`*.pyx` or `*.pxd`) have changed, the Python build must be rerun.
```

Reply:
will wait on vyasr

Reply:
There have been a number of recent and ongoing changes that will probably necessitate a bit of an overhaul, but putting this warning here is fine for now I think. Thanks for adding it!

Small note: pyd files and pxd files are not the same. pyd files are compiled extension modules on Windows. pxd files are Cython declaration files. We only care about pxd files in RAPIDS.